pennylane/devices/default_qubit.py

```diff
@@ -104,6 +104,48 @@ def accepted_sample_measurement(m: qml.measurements.MeasurementProcess) -> bool:
     )
 
 
+def null_postprocessing(results):
+    """An empty post-processing function."""
+    return results[0]
+
+
+def all_state_postprocessing(results, measurements, wire_order):
+    """Process a state measurement back into the original measurements."""
+    result = tuple(m.process_state(results[0], wire_order=wire_order) for m in measurements)
+    return result[0] if len(measurements) == 1 else result
+
+
+@qml.transform
+def adjoint_state_measurements(tape: QuantumTape) -> (Tuple[QuantumTape], Callable):
```

Comment:
I don't think we've ever really worried about this in the past, but should we be concerned with which functions render on the [device module doc page](https://xanaduai-pennylane--4865.com.readthedocs.build/en/4865/code/api/pennylane.devices.default_qubit.html#module-pennylane.devices.default_qubit)? they're in the "Internal" section, and other devices seem equally unconcerned. Still figured I'd mention it

Reply:
transforms other devices might want to use or care about.   This one is strictly default qubit.